packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -148,7 +158,42 @@ const useBigNumberConfig = (
               compact: bigNumberStyle,
           });
 
-    const showStyle = isNumber && (!isField(item) || item.format !== 'percent');
+    const formatValues = {
+        RAW: 'raw',
+        PERCENTAGE: 'percentage',
+    };
+    const [showComparison, setShowComparison] = useState<boolean>(false);
+    const [comparisonFormat, setComparisonFormat] = useState<string>(
+        formatValues.RAW,
+    );
+
+    const calculateComparisonValue = (a: number, b: number, format: string) => {
+        const rawValue = Math.round(a - b);
+        switch (format) {
+            case formatValues.PERCENTAGE:
+                return Math.round(((a - b) / b) * 100);
```

Comment:
We can call `formatValue(((a - b) / b) * 100, { format: 'percent', round: 0})` to get the formatted value.